package.json

```diff
@@ -135,7 +145,7 @@
     "build:docs": "gulp --gulpfile gulpfile.cjs docs",
     "compile": "gulp --gulpfile gulpfile.cjs compile",
     "watch": "gulp --gulpfile gulpfile.cjs watch",
-    "lint": "standard --env=mocha --env=worker",
+    "lint": "eslint src/ test/ types/",
```

Comment:
Can you add `--cache` to the lint script, and add `.eslintcache` to the `.gitignore` file? That will *hugely* speed up linting 🚀

Reply:
Oooo I did not know about that option. Nice.